contracts/src/v0.8/ccip/offRamp/OffRamp.sol

```diff
@@ -63,7 +63,8 @@ contract OffRamp is ITypeAndVersion, MultiOCR3Base {
   error ZeroAddressNotAllowed();
   error InvalidMessageDestChainSelector(uint64 messageDestChainSelector);
   error SourceChainSelectorMismatch(uint64 reportSourceChainSelector, uint64 messageSourceChainSelector);
-  error SignatureVerificationDisabled();
+  error SignatureVerificationRequired(uint8 pluginType);
```

Comment:
let's remove the args as these can only be thrown with a single arg which we can make a part of the error name

Reply:
https://github.com/smartcontractkit/chainlink/pull/14951/files#diff-865ecf8626037bea80482f856f6aff9bdd5a948982897dab6e2b4ff335af032aR66-R67